src/main/java/com/redhat/cloud/notifications/db/EmailAggregationResources.java

```diff
@@ -14,28 +14,27 @@
 public class EmailAggregationResources {
 
     @Inject
-    Mutiny.Session session;
+    Mutiny.StatelessSession statelessSession;
```

Comment:
IIRC, one of these methods is called in REST API calls - but maybe not - . Would that have a bad effect?

Reply:
After checking, all our resources are now using stateless sessions and not just the kafka ones?

Reply:
I checked that earlier and there was no call from a REST API unless I missed something. I'm in the middle of a huge rebase with tons on conflicts so I can't really check again now.

Reply:
I just confirmed `EmailSubscriptionTypeProcessor` is the only run time class calling `EmailAggregationResources` methods.